lms/routes.py

```diff
@@ -129,8 +129,8 @@ def includeme(config):  # pylint:disable=too-many-statements
     config.add_route("vitalsource_api.launch_url", "/api/vitalsource/launch_url")
 
     config.add_route("admin.index", "/admin/")
-    config.add_route("admin.instances", "/admin/instances/")
-    config.add_route("admin.instances.search", "/admin/instances/search")
+
+    config.add_route("admin.instance.search", "/admin/instances/")
```

Comment:
We seem to use the plural for this in the registrations and organizations.

Not that important but we could settle in one for all of them.

Reply:
Yes, I just didn't want to go on a rampage. The next thing I wanted to do was to potentially put these into a directory structure with the same name.

This would prevent us from having to have `/instance` and `/instances` with a single thing inside.